presto-hive/src/test/java/com/facebook/presto/hive/TestHiveSplitSource.java

```diff
@@ -257,22 +257,21 @@ public void testEmptyBucket()
     }
 
     @Test
-    public void testPreloadSplitsForGroupedExecution()
+    public void testPreloadSplitsForRewindableSplitSource()
```

Comment:
Some feedback to this test:

> assertTrue(started.await(1, SECONDS));

Usually we are trying to have higher timeouts (e.g.: 10 seconds) to avoid flaky behaviour

> List<ConnectorSplit> connectorSplits = splits.get(800, MILLISECONDS);

Ditto

Reply:
These durations follow the magnitude in `testReaderWaitsForSplits`. If you have a strong feeling I could have a separate commit to fix `testReaderWaitsForSplits` as well.

Reply:
It has been increased once https://github.com/prestodb/presto/commit/b7bd88843d763b668fd03c9876085370c6143753 (from 200 to 800). Usually to be safe if is better to keep the timeouts higher. I understand that the system scheduling delay in >1s is rather unlikely. However it is still possible. That's why i suggest keeping the timeouts higher. The only downside of higher timeouts is that you have to wait longer before the test fails.